### ISSUE TITLE
i2c-nonblocking: how to use?

### ISSUE THREAD

**Issue by jkorinth** [stm32-rs/stm32g0xx-hal#145]
What is the state of non-blocking i2c, can it be used? I tried
`cargo add stm32g0xx-hal --no-default-features --features stm32g071cb,rt,i2c-nonblocking`
but compilation fails with `could not find blocking in i2c` in `prelude.rs`.

**Comment by andresv** [comment]
Non-blocking things are under `i2c::nonblocking` https://github.com/stm32-rs/stm32g0xx-hal/blob/096121af08f320a3e115945c176c843e7526eff0/src/i2c/mod.rs#L5

PS: have you looked into [embassy](https://embassy.dev/). It has the most comprehensive driver support for all STM32 families.
